### PR TITLE
Add components and component groups from aframe-extras

### LINKED ISSUE #90: Individually mention each component provided by aframe-extras
By @donmccurdy

### PR DESCRIPTION
**Pull request by donmccurdy**
Includes the bits that seem worthwhile, anyway.

Fixes #90.

### CODE REVIEW

**Comment by donmccurdy** [comment]
Should I remove the separate aframe-extras reference?

File: README.md [hunk 1]
```diff
@@ -108,27 +108,32 @@ Third-party components (of the entity-component-system pattern).
 - [Mesh Line](https://github.com/andreasplesch/aframe-meshline-component) - Draw thick lines
 - [Mouse Cursor](https://github.com/mayognaise/aframe-mouse-cursor-component) - Use mouse as a direct pointer
 - [No-click Look Controls](https://github.com/alexrkass/aframe-no-click-look-controls) - Intuitive desktop view controls without requiring mousedown + drag
-- [OBJ and MTL Loader](https://github.com/donmccurdy/aframe-obj-loader-component) - Loader for .OBJ models and .MTL materials
 - [Ocean](https://github.com/donmccurdy/aframe-extras/blob/master/src/primitives/a-ocean.js) - Flat-shaded ocean primitive and component, with animated waves
 - [Orbit Controls](https://github.com/subsumo/aframe-orbit-controls/) - Orbit camera around an entity
-- [Physics](https://github.com/ngokevin/aframe-physics-components) - Rigid-body physics
+- [Physics (ngokevin)](https://github.com/ngokevin/aframe-physics-components) - Rigid-body physics using Cannon.js
```
Comment (ngokevin) [inline README.md]: Let's just remove mine.

**Comment by ngokevin** [comment]
I'd add some controls like checkpoint and mention pointer lock in case someone searches for that

**Comment by donmccurdy** [comment]
checkpoint is there, if a bit hard to find. going to need a JSON manifest and search/filters at some point here :)

**Comment by donmccurdy** [comment]
^best kind of problem to have.

**Comment by ngokevin** [comment]
Hehe, I think I just didn't search at all.

We will be working heavily with component discovery and consumption next cycle. Would probably include a proper frontend to all these neat components.

**Comment by ngokevin** [comment]
Thanks!

**Pull request merged**